internal/grpc_dialer.go

```diff
@@ -78,11 +79,21 @@ func dial(params dialParameters) (*grpc.ClientConn, error) {
 	cp.Backoff.BaseDelay = retryPollOperationInitialInterval
 	cp.Backoff.MaxDelay = retryPollOperationMaxInterval
 
+	// gRPC utilizes keep alive mechanism to detect dead connections in case if server didn't close them
+	// gracefully. Client would ping the server periodically and expect replies withing the specified timeout.
```

Comment:
".. in the case that the server didn't .."

Reply:
fixed